### ISSUE TITLE
Command to change which cursor is primary

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#135460]
When a user adds multiple cursors we nicely announce the locations for each additional cursors.
However the user can not check what is going on at the second / third cursor.

I thought it would be nice if the user could change which one is the primary cursor using a command. That way the user could  traverse each of the cursors and be able to read the line on each cursor for example. Since for screen reader reading we only respect the primary cursor.

Now I am not sure if this is even possible from the implementation. Thus I am creating this issue for discussion.
I am also open for ideas.

**Comment by isidorn** [comment]
Thanks a lot for pushing this @hediet 👏 

Once this is in insiders tomorrow I will do a push in our accessibility community for multi cursor, since I think with this it should be much more usable. So we get feedback.
Did you verify if you change the focus of the cursor, does the screen reader read the new line (where the newly focused cursor is). I believe it should just work.

**Comment by hediet** [comment]
Verification steps:
* Launch nvda
* Set multiple cursors
* Use "Focus Next Cursor" / "Focus Previous Cursor"
* Verify that NVDA reads the text of the primary cursor and that the primary cursor rotates
